crates/tabby-inference/src/lib.rs

```diff
@@ -24,6 +24,17 @@ pub struct TextGenerationOptions {
     pub language: Option<&'static Language>,
 }
 
+impl TextGenerationOptions {
+    pub const DEFAULT_TEMPERATURE: f32 = 0.1;
```

Comment:
do we still need to have this since we set default value to be 0.1 in `builder`?

You might remove it and cleanup the code in service